libs/libzip/Makefile

```diff
@@ -6,12 +6,12 @@
 include $(TOPDIR)/rules.mk
```

Comment:
This shouldn't be in this PR at all.

Reply:
Doh. Not sure how that got there.  Rebased.